src/Controller/Backend/LeadDetailsController.php

```diff
@@ -77,6 +78,10 @@ public function __invoke(int $id)
 
         $formData = $this->getFormData($id);
         $languages = $this->systemAdapter->getLanguages();
+        $locale = $formData->language;
+        if (version_compare(VERSION, '4.13', '>=')) {
+            $locale = LocaleUtil::formatAsLocale($locale);
+        }
```

Comment:
```suggestion
        $locale = $formData->language;

        if (class_exists(LocaleUtil)) {
            $locale = LocaleUtil::formatAsLocale($locale);
        }
```
I think the version constant is deprecated.

Reply:
First I added the `version_compare` and then I used `LocaleUtil` - after that today I also thought a `class_exists` might make more sense 😁